doc/040_backup.rst

```diff
@@ -213,13 +213,26 @@ On most Unixy shells, you can either quote or use backslashes. For example:
 
 By specifying the option ``--one-file-system`` you can instruct restic
 to only backup files from the file systems the initially specified files
-or directories reside on. For example, calling restic like this won't
-backup ``/sys`` or ``/dev`` on a Linux system:
+or directories reside on. So it will prevent restic from crossing filesystem 
+boundaries when performing a backup. For example, if you backup ``/`` with this 
+flag but you have external media mounted under ``/media/usb`` then restic will 
+not back up ``/media/usb`` at all because it *is* a different filesystem.
+Also calling restic on ``/`` would not descend into any virtual
+filesystems so it won't backup ``/sys`` or ``/proc`` on a Linux system:
```

Comment:
Please change these lines to the following:

or directories reside on. In other words, it will prevent restic from crossing filesystem boundaries when performing a backup.

For example, if you backup ``/`` with this option and you have external media mounted under ``/media/usb`` then restic will not back up ``/media/usb`` at all because this is a different filesystem than ``/``. Virtual filesystems such as ``/proc`` are also considered different and thereby excluded when using ``--one-file-system``: